lib/puppet/type/netscaler_service_lbmonitor_bind.rb

```diff
@@ -25,9 +25,9 @@
     truthy_property('The configured state (enable/disable) of the bound monitor.','ENABLED','DISABLED')
   end
 
-  newproperty(:passive, :parent => Puppet::Property::NetscalerTruthy) do
-    truthy_property('Indicates if the monitor is passive. A passive monitor does not remove service from LB decision when the threshold is breached.','ENABLED','DISABLED')
-  end
+  # newproperty(:passive, :parent => Puppet::Property::NetscalerTruthy) do
```

Comment:
Could you comment about why this is commented out, so it doesn't just get deleted?